recipe/meta.yaml

```diff
@@ -43,7 +43,10 @@ outputs:
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
       host:
         - cuda-version {{ cuda_version }}
+        - expat
         - gmp
+        - mpfr
+        - ncurses
```

Comment:
Saw this error [on CI]( https://app.travis-ci.com/github/conda-forge/cuda-gdb-feedstock/jobs/603472327#L620 )

```
  ERROR (cuda-gdb,bin/cuda-gdb): $RPATH/libtinfo.so.5 not found in packages, sysroot(s) nor the missing_dso_whitelist.
```

This only happens on `ppc64le`. Though added `ncurses` generally in case it is actually needed for other architecture packages as well